x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -102,7 +102,7 @@ export const AlertsTableComponent: FC<DetectionEngineAlertTableProps> = ({
 }) => {
   const { triggersActionsUi, uiSettings } = useKibana().services;
 
-  const { from, to, setQuery } = useGlobalTime();
+  const { from, to, setQuery } = useGlobalTime(false);
```

Comment:
`useGlobalTime` calls `deleteQuery` method on dismount. Passing `false` disables this behavior. 

This fixes the bug, `Closing an alert group triggers an unnecessary update/fetch on the trends graph`. So now the queries can build up like this, but when you navigate away from the alerts page, they are cleared again.
<img width="757" alt="Screenshot 2023-04-19 at 8 34 59 AM" src="https://user-images.githubusercontent.com/6935300/233092647-96d43e08-c163-4ee5-86a8-29611c9dba4d.png">

Does adding false cause issues elsewhere? Not that I've found or the tests have found. I am not sure when it matters to clear the queries, or if it matters. Why are we doing this? What is the harm of leaving the queries? Should `useGlobalTime`'s `clearAllQuery` argument be defaulting to `false` instead of `true`?

I see in the git blame @XavierM, I will ask him and ask around other threat hunting engineers.